resources/lang/fr/widgets.php

```diff
@@ -16,7 +16,7 @@
     ],
 
     'weather' => [
-        'nometar'    => 'METAR/TAF n\'ont pas pu être récupérés',
+        'nometar'    => 'le METAR/TAF n\'a pas pu être récupéré',
```

Comment:
Le METAR/TAF (uppercase for the first letter of the sentence)

Reply:
Right